slaves-template.yml

```diff
@@ -0,0 +1,215 @@
+          kind: ImageStreamTag
+          name: "${JENKINS_SERVICE_NAME}-slave-java:latest"
+      runPolicy: Serial
+      source: 
+        contextDir: slave-java
+        git: 
+          ref: "${GITHUB_REF}"
+          uri: "https://github.com/${GITHUB_ORG}/wendy-jenkins-s2i-configuration"
+        type: Git
+      strategy: 
+        dockerStrategy: 
+          noCache: true
+        type: Docker
+      triggers: 
+        - 
+          type: ImageChange
+        - 
+          type: ConfigChange
+  - 
+    apiVersion: v1
+    kind: ImageStream
+    metadata: 
+      annotations: 
+        slave-label: ruby
+      labels: 
+        role: jenkins-slave
+      name: "${JENKINS_SERVICE_NAME}-slave-ruby"
+  - 
+    apiVersion: v1
+    kind: BuildConfig
+    metadata: 
+      name: "${JENKINS_SERVICE_NAME}-slave-ruby"
+    spec: 
+      output: 
+        to: 
+          kind: ImageStreamTag
+          name: "${JENKINS_SERVICE_NAME}-slave-ruby:latest"
+      runPolicy: Serial
+      source: 
+        contextDir: slave-ruby
+        git: 
+          ref: "${GITHUB_REF}"
+          uri: "https://github.com/${GITHUB_ORG}/wendy-jenkins-s2i-configuration"
+        type: Git
+      strategy: 
+        dockerStrategy: 
+          noCache: true
+        type: Docker
+      triggers: 
+        - 
+          type: ImageChange
+        - 
+          type: ConfigChange
+  - 
+    apiVersion: v1
+    kind: ImageStream
+    metadata: 
+      annotations: 
+        slave-label: ruby-fhcap
+      labels: 
+        role: jenkins-slave
+      name: "${JENKINS_SERVICE_NAME}-slave-ruby-fhcap"
+  - 
+    apiVersion: v1
+    kind: BuildConfig
+    metadata: 
+      name: "${JENKINS_SERVICE_NAME}-slave-ruby-fhcap"
+    spec: 
+      output: 
+        to: 
+          kind: ImageStreamTag
+          name: "${JENKINS_SERVICE_NAME}-slave-ruby-fhcap:latest"
+      runPolicy: Serial
+      source: 
+        contextDir: slave-ruby-fhcap
+        git: 
+          ref: "${GITHUB_REF}"
+          uri: "https://github.com/${GITHUB_ORG}/wendy-jenkins-s2i-configuration"
+        type: Git
+      strategy: 
+        type: Docker
+      triggers: 
+        - 
+          type: ImageChange
+        - 
+          type: ConfigChange
+  -
+    apiVersion: v1
+    kind: "LimitRange"
+    metadata:
+      name: jenkins-resource-limits
+    spec:
+      limits:
+        - type: Container
+          defaultRequest:
+            cpu: 1
+            memory: 1Gi
+parameters: 
+  - 
+    description: "The name of the OpenShift Service exposed for the Jenkins container."
+    displayName: "Jenkins Service Name"
+    name: JENKINS_SERVICE_NAME
+    value: jenkins
+  - 
+    description: "The name of the github organization to reference in the configuration"
+    displayName: "Github Organization"
+    name: GITHUB_ORG
+    value: feedhenry
+  - 
+    description: "The name of the repository branch to reference in the configuration"
+    displayName: Branch
+    name: GITHUB_REF
+    value: master
```

Comment:
Assuming this is just copied directly with no changes from the jenkins-template, it looks fine.